docs/content/configuration/creating_pvc.md

```diff
@@ -42,7 +42,7 @@ Create a PVC YAML file, similar to the following `demo-pvc-file-system.yaml` fil
           storage: 1Gi
       storageClassName: demo-storageclass
 
-## <a name=PVC-block></a>Creating PVC for raw block volume
+## Creating PVC for raw block volume
```

Comment:
btw, why is it `Creating a ...` in the main title, but `Creating PVC` (without `a`) in the sub-titles?
I think an indefinite article is needed in all of them

Reply:
updated - DONE (+ fixed linking)